src/cdk/constructs/domain-manager/resource/route53-manager.ts

```diff
@@ -1,103 +1,110 @@
-import { CfnOutput, Construct, Duration } from '@aws-cdk/core'
-import { Topic } from '@aws-cdk/aws-sns'
-import { SingletonFunction, Runtime, AssetCode } from '@aws-cdk/aws-lambda'
-import { FollowMode } from '@aws-cdk/assets'
-import { SnsEventSource } from '@aws-cdk/aws-lambda-event-sources'
 import { AccountPrincipal, CompositePrincipal, Effect, ManagedPolicy, PolicyStatement, Role, ServicePrincipal } from '@aws-cdk/aws-iam'
-import { MiraConfig } from '../../../../config/mira-config'
+import { Code, Runtime, SingletonFunction } from '@aws-cdk/aws-lambda'
+import { SnsEventSource } from '@aws-cdk/aws-lambda-event-sources'
+import { Topic } from '@aws-cdk/aws-sns'
+import { CfnOutput, Construct, Duration } from '@aws-cdk/core'
 import path from 'path'
+import { MiraConfig } from '../../../..'
 import { MiraStack } from '../../../stack'
+import { ROUTE53_MANAGER_POLICY_BOUNDARY } from './route53-manager-access-role'
+
+export const DOMAIN_SUBSCRIPTION_TOPIC = 'DomainSubscriptionTopic'
```

Comment:
Do we maybe want a consts file for string constants?

Reply:
I'm strongly against one centralized consts file and prefer to have consts declared where they logically belong.

Reply:
Ok.. i think since this is a SNS topic this should be fine as it'll scale to whatever horizontal scaling needs are.  Can we document the topic in the docs and what subscribers to with published messages to the topic?

Reply:
I've added it in the latest change